akka-stream-typed/src/test/scala/docs/akka/stream/typed/ActorSourceSinkExample.scala

```diff
@@ -44,7 +45,10 @@ object ActorSourceSinkExample {
     // #actor-source-ref
   }
 
+
   {
+    def targetActor(): ActorRef[Protocol] = ???
```

Comment:
not a warning here?

Reply:
It was a dead code warning for the code below the ???, i've now removed dead code warnings from ytest